plugins/inventory/scaleway.py

```diff
@@ -278,13 +290,32 @@ def do_zone_inventory(self, zone, token, tags, hostname_preferences):
                 # Composed variables
                 self._set_composite_vars(self.get_option('variables'), host_infos, hostname, strict=False)
 
+    def get_oauth_token(self):
+        oauth_token = self.get_option('oauth_token')
+        scw_config = os.path.join(os.path.expanduser('~'), '.config', 'scw', 'config.yaml')
```

Comment:
The CLI and other Scaleway devtools choose the config path in the following way (cf. [this doc](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#config-file-path)):

>     1 .Custom directory: $SCW_CONFIG_PATH
>     2. XDG base directory: $XDG_CONFIG_HOME/scw/config.yaml
>     3. Unix home directory: $HOME/.config/scw/config.yaml
>     4. Windows home directory: %USERPROFILE%/.config/scw/config.yaml

Can we make it consistent in this MR too?

Reply:
Having a function `find_config_file` which returns its path or `None` if it can't find it is probably a good idea. In any case, we can ignore case 4 since this collection does not support Windows targets.

Reply:
`os.path.expanduser('~')` expand to `%USERPROFILE%` on Windows, so that's not an issue.